src/components/App.jsx

```diff
@@ -37,7 +37,7 @@ export const App = () => {
       <RealTimeQueries doctype="io.cozy.oauth.clients" />
       <SettingsRealTimeQueries />
       <Main>
-        <Content className="u-p-1">
+        <Content>
```

Comment:
no ui issue on mobile?

Reply:
Found nothing after trying desktop and mobile.